gme/blargg_common.h

```diff
@@ -27,10 +27,11 @@
 	#define STATIC_CAST(T,expr) ((T) (expr))
 #endif
 
-#if !defined(_MSC_VER) || _MSC_VER >= 1910
-	#define blaarg_static_assert(cond, msg) static_assert(cond, msg)
+#if defined(_MSC_VER) && _MSC_VER < 1910
+	#define blaarg_static_assert(cond, msg) \
+	        void blargg_failed_(int (*arg) [2 / (int) !!(cond) - 1])
```

Comment:
I don't know, but I never liked the `!!` way to turn into the boolean, I prefer the explicit `(bool)` cast. Anyway, I bet, double-negation takes more instructions than the casting.

Reply:
It's a compile-time calculation  (and I actually like `!!` :)